client/boot/index.js

```diff
@@ -295,7 +295,7 @@ function reduxStoreReady( reduxStore ) {
 		next();
 	} );
 
-	if ( config.isEnabled( 'oauth' ) ) {
+	if ( config.isEnabled( 'oauth' ) && config.isEnabled( 'desktop' ) ) {
```

Comment:
Do we really want this change?
I tried loading calypso web at http://calypso.localhost:3000/, the app loaded on the main page but all xhr requests failed without showing me any error on the app.
I'm expecting to be redirected to the login page, why don't we want this for the web app?